WalletWasabi/WabiSabi/Backend/Rounds/Round.cs

```diff
@@ -65,6 +63,9 @@ public Round(RoundParameters roundParameters)
 		public int InitialInputVsizeAllocation { get; set; } = 99954; // TODO compute as CoinjoinState.Parameters.MaxWeight - CoinjoinState.Parameters.SharedOverhead, mutable for testing until then
 		public int RemainingInputVsizeAllocation => InitialInputVsizeAllocation - Alices.Count * (int)PerAliceVsizeAllocation;
 
+		private RoundParameters RoundParameters { get; }
+		private WasabiRandom Random => RoundParameters.Random;
```

Comment:
for get status requests we probably want to refactor `RoundParameters` to be serializable and shared by the client & server, IIRC from #5457 `Random` was the only property that prevented that so it should probably be a constructor parameter to `Round` instead of a part of `RoundParameters` (and relatedly, a `MultipartyTransactionParameters` property should be added to `RoundParameters` it as well, I've been dragging my feet on that, sorry...)

Reply:
Ok, that's for another PR. I removed the `Random` property btw.